### ISSUE TITLE
Add User Timeline section

### ISSUE THREAD

**Issue by GoogleCodeExporter** [emuikernel/wittytwitter#1]
```
Clicking on the username should display the specific tweets for that user
within Witty.  Currently it links to the user's twitterpage.


```

Original issue reported on code.google.com by `alan...@gmail.com` on 17 Oct 2007 at 8:24

**Comment by GoogleCodeExporter** [comment]
Original comment by `alan...@gmail.com` on 17 Oct 2007 at 10:17
- Added labels: **Milestone-v0.1.3**

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
implemented for v0.1.3
```

Original comment by `alan...@gmail.com` on 18 Oct 2007 at 9:27
